src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -34,10 +34,29 @@
 * @description Installment Opportunity Evaluation and Processing
 *
 */
-public class RD2_OpportunityEvaluationService {
+public inherited sharing class RD2_OpportunityEvaluationService {
 
     private npe03__Recurring_Donations_Settings__c rdSettings = UTIL_CustomSettingsFacade.getRecurringDonationsSettings();
 
+    /**
+    * @description Flag to allow customers to optionally disable Rollups before inserting/updating Opps
+    * to address issues with performance in larger organizatios
+    */
+    @TestVisible
+    private Boolean disableRollupsBeforeDml {
```

Comment:
Good point. In a very near future WI (one that's actually not specific to RD2), we will introduce this new field on the RD Settings object. At that point, we could just address it in the code as `rdSettings.DisableRollupsInBatchJob__c`. That doesn't exist yet though, so I've separated into this getter method.

Reply:
The var can be local variable in the `disableRollups()`.

Reply:
True. I've been thinking about the concept of not adding placeholders for upcoming work. How about if I just remove this and add a simple `// TODO` comment to reflect that this will be needed in the near future?

Reply:
This method is fine, I do not see any conflicts with future work. It would be great if we could localize changes, ie the `if` statement and the class variable related to the method pulled in, otherwise, I vote for this method and its call from `execute()` to stay. 
Other TODOs are ok as long as they will be addressed in future WIs.
The only merge conflict I see is work related to Close RD open Opportunities and the rename of `isCandidate()`.